knowledge-base/upload-custom-validation-error.md

```diff
@@ -0,0 +1,79 @@
+---
+title: Create a custom validation error
+description: An example on how to create a custom validation error for the Kendo UI for Vue Native Upload
+type: how-to
+page_title: Create a custom validation error - Kendo UI for Vue Native Upload
+slug: upload-custom-validation-error
+tags: upload, kendovue, validation, error, custom
+ticketid: 
+res_type: kb
+category: knowledge-base
+---
+
+## Environment
+
+<table>
+    <tbody>
+	    <tr>
+	    	<td>Product Version</td>
+	    	<td>4.3.1</td>
+	    </tr>
+	    <tr>
+	    	<td>Product</td>
+	    	<td>Progress® Kendo UI for Vue Native Upload</td>
+	    </tr>
+    </tbody>
+</table>
+
+## Description
+
+This Knowledge base(KB) article demonstrates how we can add a custom validation error for the Upload component.
```

Comment:
Should we a sentence explaining when the added custom error will be shown in the demo?